src/screens/App.tsx

```diff
@@ -1,23 +1,95 @@
-import { useState } from "react";
-
+import {
+  ButtonLogin,
+  ButtonCreate,
+  ButtonRegister
+} from "../components/Button/Button";
+import { FeatureTitle, FeatureDescription } from "../components/Text/Text";
 export function App() {
-  const [count, setCount] = useState(0);
-
   return (
-    <main className="flex h-screen w-full flex-col items-center justify-center gap-2 bg-black">
-      <h1 className="text-xl text-white">
-        Vite + React + Typescript + TailwindCSS | Hi
-      </h1>
-      <button
-        type="button"
-        className="bg-white p-2 text-black"
-        onClick={() => {
-          setCount((count) => count + 1);
-        }}
-      >
-        Increase
-      </button>
-      <p className="text-white">Count: {count}</p>
+    <main className="h-screen w-full">
+      <div className="min-h-1/2 bg-blue-600 pb-16">
+        <div className=" mx-auto max-w-screen-2xl">
+          <div className="left-4 top-4 flex items-center font-bold text-white sm:left-8 md:left-12 lg:left-16 xl:left-20">
```

Comment:
Positions (`left`, `right`, `top`, `bottom`) doesn't have any effect on this `div`, also, avoid using every media query (`sm:`, `md:`, `lg:`, `xl:`)